packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -299,6 +312,9 @@ const filteredHeaders = computed(() => {
     ProjectName.TFRobot,
     ProjectName.Gitea,
     ProjectName.Nostr,
+    ProjectName.Algorand,
+    ProjectName.Subsquid,
+    ProjectName.Peertube,
```

Comment:
IPV4 also?

Reply:
IPV6 and IPV4 were added to all solutions, therefore I added them to the deployment lists of all solutions